.github/workflows/windows.yml

```diff
@@ -23,9 +21,20 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-vcpkg-download-${{ matrix.os }}-
             ${{ runner.os }}-vcpkg-download-
+      - name: Set VCVARS
+        run: |
+          IF ${{ matrix.os }}==windows-2022 (
+            echo VCVARS="C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Auxiliary\Build\vcvars64.bat" >> %GITHUB_ENV%
+          ) ELSE IF ${{ matrix.os }}==windows-2019 (
+            echo VCVARS="C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat" >> %GITHUB_ENV%
```

Comment:
This makes `%VCVARS%` to include double quotes.
It feels weird a little.

Reply:
PATH without double quotes causes an error.
So I add double quotes to `echo` again, and remove from `call`.

https://github.com/ruby/ruby/runs/3418598073?check_suite_focus=true#step:4:11
```
Run IF windows-2022==windows-2022 (
  IF windows-2022==windows-2022 (
    echo VCVARS=C:\Program Files\Microsoft Visual Studio\2022\Preview\VC\Auxiliary\Build\vcvars64.bat >> %GITHUB_ENV%
  ) ELSE IF windows-2022==windows-2019 (
    echo VCVARS=C:\Program Files (x86)\Microsoft Visual Studio\2019\Enterprise\VC\Auxiliary\Build\vcvars64.bat >> %GITHUB_ENV%
  )
  shell: C:\Windows\system32\cmd.EXE /D /E:ON /V:OFF /S /C "CALL "{0}""
  env:
    GITPULLOPTIONS: --no-tags origin refs/pull/4775/merge
    PATCH: C:\msys64\usr\bin\patch.exe
\Microsoft was unexpected at this time.
Error: Process completed with exit code 255.
```